baseplate/experiments/__init__.py

```diff
@@ -172,54 +194,65 @@ def variant(self, name, user=None, bucketing_event_override=None,
         if do_log:
             assert bucketing_id
             self._log_bucketing_event(experiment, variant, user,
-                                      extra_event_fields)
+                                      inputs)
             self._already_bucketed.add(bucketing_id)
 
         return variant
 
-    def _log_bucketing_event(self, experiment, variant, user,
-                             extra_event_fields):
-        if not self._event_queue:
-            return
-
-        extra_event_fields = extra_event_fields or {}
+    def _log_bucketing_event(self, experiment, variant, user, extra_event_fields):
+
+        """ Logs a v2 bucketing event.
```

Comment:
style nit: [PEP-257](https://github.com/reddit/styleguide/blob/master/python/pep-0257.rst) says `The docstring is a phrase ending in a period. It prescribes the function or method's effect as a command ("Do this", "Return that"), not as a description; e.g. don't write "Returns the pathname ...".`